app/controllers/posts_controller.rb

```diff
@@ -28,6 +29,12 @@ def create
     end
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+    @post.author.decrement!(:posts_counter)
+    Post.delete(params[:id])
+  end
+
```

Comment:
- [ ] [OPTIONAL] it would be better when you delete a post you should redirect to last page in order I know that post has been deleted successfully with a notice like this `redirect_to (the path), notice: deleted succesfully `

Reply:
@youmari I think I managed the fix, but I do not know what is happening.